embuilder.py

```diff
@@ -128,6 +128,7 @@ def main():
                                    formatter_class=argparse.RawDescriptionHelpFormatter,
                                    epilog=get_help())
   parser.add_argument('--lto', action='store_true', help='build bitcode object for LTO')
+  parser.add_argument('--thinlto', action='store_true', help='build bitcode object for ThinLTO')
```

Comment:
Can we add a check (maybe after line 145) that reports an error if both LTO and thinLTO are specified together? (the current behavior just silently lets thinlto "win")

Reply:
I can add a check like:

```python
  if args.lto and args.thinlto:
      logger.error('invalid flags `--lto --thinlto` specified, cannot use both.')
      return 1
```

I am just thinking this looks kinda dirty, like a checkbox to work like a radio button, trying to think if there's some design that still works with argparse and is still backwards compatible. Otherwise the flag could be just `--thin` and instead be invalid if using without `--lto`.

Reply:
I made a new idea, that can support flag if it has no spaces with argparse.